src/pudl/package_data/glue/utility_id_pudl.csv

```diff
@@ -15942,3 +15942,207 @@ utility_id_pudl,utility_id_ferc1,utility_name_ferc1,utility_id_eia,utility_name_
 15984,,,65828,Santa Barbara County
 15985,,,65834,DE Shaw Renewable Investments
 15986,,,65850,"Hull Street Energy, LLC"
+15987,,,65851,"Omnis Pleasants, LLC"
```

Comment:
Non-blocking: is there a reason some names are in quotations and others aren't?

Reply:
That's just how I copied them from the unmapped_utility_id spreadsheet....I'll double check that's how they look in the output, but seems odd!

Reply:
When I open up the spreadsheet I don't see any quotation marks...I'm not sure why they are showing up here.

Reply:
Oh duh, it's the ones that have commas in the name because it's a comma-separated value table. All good, as expected and nothing to see here!

Reply:
I did find out that I had double pasted the utility id values though!! Removing the duplicates.